src/photon/photon_scheduler.c

```diff
@@ -162,8 +162,14 @@ void reconstruct_object_task_lookup_callback(object_id reconstruct_object_id,
                                              task *task,
                                              void *user_context) {
   /* Recursively resubmit the task and its task lineage to the scheduler. */
-  CHECKM(task != NULL,
-         "No task information found for object during reconstruction");
+  /* TODO(swang): The following check will fail if an object was created by a
+   * put. Once we're able to reconstruct those objects as well, uncomment the
+   * check and remove the if-check and exit. */
+  /* CHECKM(task != NULL,
+         "No task information found for object during reconstruction"); */
+  if (task == NULL) {
```

Comment:
We can reconstruct puts, but the rest of the code that's needed for that isn't in place yet. I was hoping to do those in another PR.